control/modelsimp.py

```diff
@@ -427,11 +439,22 @@ def markov(Y, U, m=None, transpose=False):
         time points.
     U : array_like
         Input data, arranged in the same way as `Y`.
+    data : TimeResponseData
+        Response data from which the Markov parameters where estimated.
+        Input and output data must be 1D or 2D array.
     m : int, optional
         Number of Markov parameters to output.  Defaults to len(U).
+    dt : True of float, optional
+        True indicates discrete time with unspecified sampling time,
+        positive number is discrete time with specified sampling time.It
+        can be used to scale the markov parameters in order to match the
```

Comment:
In the case that a TimeResponseData object is given, should `dt` default to the value coming from the time array?  You could set the default value of `dt` to `None` and then if the input is `Y, U` then you default to `dt=1` and the input is data, you default to `data.time[1] - data.time[0]` (perhaps with a check to make sure that time points are equally spaced?).

Reply:
Done.